tests/system/test_metadata.py

```diff
@@ -229,6 +229,36 @@ class AnyKind(ndb.Model):
     assert properties == ["foo"]
 
 
+@pytest.mark.usefixtures("client_context")
+@pytest.mark.parametrize("namespace", ["DiffNamespace"])
```

Comment:
This spelling looks wrong. I've had tricky issues before where pytest accepts anything after "mark". If it's not the right thing, you don't get an error, it just ignores you. I had "@pytest.mark.use_fixtures" on something and it took a while to figure out why my fixture wasn't being used, as a result.

Point being, I'd double check that you're actually testing what you think you are here.

Reply:
You mean "parametrize"? Looks funny to me too, but that's the way the pytest author writes it.